Documentation/ApiOverview/CommandControllers/ListCommands.rst

```diff
@@ -0,0 +1,34 @@
+:navigation-title: List Core commands
+..  include:: /Includes.rst.txt
+
+..  _symfony-console-commands-list:
+
+==============================
+List of TYPO3 console commands
+==============================
+
+..  todo: List all common commands once https://forge.typo3.org/issues/103858 is resolved
+
+By default TYPO3 ships the listed console commands, depending
+on which system extensions are installed.
+
+Third party extensions can define :ref:`custom console
+commands <console-command-tutorial>`.
+
+The extension :t3ext:`typo3_console` ships many commands to execute TYPO3
+actions, which otherwise would only be accessible via the TYPO3 backend.
+
+This page assumes that the code is run on a Composer based installation with
+default binaries location. Here you can read how to run them in general and
+on legacy installations:
+:ref:`Run a command from the command line <symfony-console-commands-cli>`.
+
+..  _symfony-console-commands-list-list:
```

Comment:
```suggestion
..  _symfony-console-commands-list:
```

Reply:
"symfony-console-commands-list" is alread used above. this is a list of commands, listing the command list...